src/main/java/org/fisco/bcos/web3j/precompile/frozen/Frozen.java

```diff
@@ -0,0 +1,333 @@
+                        Arrays.<TypeReference<?>>asList(new TypeReference<Address>() {}));
+        List<Type> results = FunctionReturnDecoder.decode(data, function.getOutputParameters());
+        ;
+        return new Tuple1<String>((String) results.get(0).getValue());
+    }
+
+    public Tuple1<BigInteger> getFreezeOutput(TransactionReceipt transactionReceipt) {
+        String data = transactionReceipt.getOutput();
+        final Function function =
+                new Function(
+                        FUNC_FREEZE,
+                        Arrays.<Type>asList(),
+                        Arrays.<TypeReference<?>>asList(new TypeReference<Int256>() {}));
+        List<Type> results = FunctionReturnDecoder.decode(data, function.getOutputParameters());
+        ;
+        return new Tuple1<BigInteger>((BigInteger) results.get(0).getValue());
+    }
+
+    public RemoteCall<Tuple2<BigInteger, String>> queryStatus(String addr) {
+        final Function function =
+                new Function(
+                        FUNC_QUERYSTATUS,
+                        Arrays.<Type>asList(new org.fisco.bcos.web3j.abi.datatypes.Address(addr)),
+                        Arrays.<TypeReference<?>>asList(
+                                new TypeReference<Uint256>() {},
+                                new TypeReference<Utf8String>() {}));
+        return new RemoteCall<Tuple2<BigInteger, String>>(
+                new Callable<Tuple2<BigInteger, String>>() {
+                    @Override
+                    public Tuple2<BigInteger, String> call() throws Exception {
+                        List<Type> results = executeCallMultipleValueReturn(function);
+                        return new Tuple2<BigInteger, String>(
+                                (BigInteger) results.get(0).getValue(),
+                                (String) results.get(1).getValue());
+                    }
+                });
+    }
+
+    public RemoteCall<TransactionReceipt> kill(String addr) {
+        final Function function =
+                new Function(
+                        FUNC_KILL,
+                        Arrays.<Type>asList(new org.fisco.bcos.web3j.abi.datatypes.Address(addr)),
+                        Collections.<TypeReference<?>>emptyList());
+        return executeRemoteCallTransaction(function);
+    }
+
+    public void kill(String addr, TransactionSucCallback callback) {
+        final Function function =
+                new Function(
+                        FUNC_KILL,
+                        Arrays.<Type>asList(new org.fisco.bcos.web3j.abi.datatypes.Address(addr)),
+                        Collections.<TypeReference<?>>emptyList());
+        asyncExecuteTransaction(function, callback);
+    }
+
+    public String killSeq(String addr) {
+        final Function function =
+                new Function(
+                        FUNC_KILL,
+                        Arrays.<Type>asList(new org.fisco.bcos.web3j.abi.datatypes.Address(addr)),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.fisco.bcos.web3j.abi.datatypes.Address' due to existing import 'org.fisco.bcos.web3j.abi.datatypes.Address'](https://app.codacy.com/manual/fisco/web3sdk/pullRequest?prid=4981260)